gaj23/face-it
# Kby feature error handling

#### Purpose: 
> Check all that apply.

### What does this PR do? 
> Clearly and concisely describe changes made. 
- CSS - alphabetize CSS style properties
- Components - change double quotes to single quotes
- add error handling for components


### Problems Encountered & Resulting Solutions  
> State any problems encountered 
> Describe your code changes in detail for reviewers.
- CSS - alphabetize CSS style properties
- Components - change double quotes to single quotes
- add error handling for components


### Where should the reviewer start?
> What file(s) and line(s) are where changes are made?
- CSS - alphabetize CSS style properties
- Components - change double quotes to single quotes
- add error handling for components


### How should this be tested?
> What steps need to be taken to confirm functionality? 
npm start => browser window

### Steps To Take
> Any questions and possible steps moving forward.



#### Checklist:
